Lib/unittest/mock.py

```diff
@@ -705,6 +714,11 @@ def __setattr__(self, name, value):
         else:
             if _check_and_set_parent(self, value, name, name):
                 self._mock_children[name] = value
+
+        if self._mock_sealed and not getattr(self, name):
```

Comment:
Did you want to use `hasattr` here? `getattr(self, name)` may raise an exception.

Reply:
done